src/Data/Formatter/Number.purs

```diff
@@ -63,6 +74,8 @@ instance showFormatter :: Show Formatter where
 
 derive instance eqFormatter :: Eq Formatter
 
+-- | The format string representation of a `Formatter`.
+-- | The interpretation of the format string inspired by [numeral.js](http://numeraljs.com/#format)
```

Comment:
Same thing applies here as to the comment about moment.js. Also this information is news to me, I didn't know where it came from :smile: (I didn't implement this module).